crypto/rand/drbg_lib.c

```diff
@@ -16,6 +16,8 @@
 #include "internal/rand_int.h"
 #include "internal/cryptlib_int.h"
 
+#define HACK(n,x) { size_t _i, _n = (n); const unsigned char *_x = x; \
```

Comment:
I don't understand what this is doing????
Well I do but don't understand why.

Reply:
I will back out that again, but currently the msan shows AES using uninitilaized data
in the not-constant time part of the code.

Reply:
Something weird is going on.  This isn't writing to memory and shouldn't be able to initialise anything.

Reply:
I want to track down where the the heck the uninitialized data is coming from.
I will be back in an hour or so.